ci/prow/boskos/make_custom_role.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CUSTOM_ROLE="custom_role.yaml"
+CUSTOM_ROLE_PERMS=/tmp/custom_role_permissions.yaml
+
+rm -f ${CUSTOM_ROLE_PERMS}
+for r in editor storage.admin; do
```

Comment:
For safety, I give it all permissions owned by `editor` and `storage.admin`. We can make followup PRs for more fine-grained access control.

Reply:
sgtm